tests/Integration/Behaviour/Features/Scenario/Product/duplicate_product.feature

```diff
@@ -31,6 +31,14 @@ Feature: Duplicate product from Back Office (BO).
       | name[en-US] | Reading glasses |
       | name[fr-FR] | lunettes        |
       | type        | standard        |
+    Given I add product "product3" with following information:
```

Comment:
You should create this pack product only for your pack product scenario It's not needed in other scenarios

Reply:
I don't understand .... can we discuss about that

Reply:
done